test/common/router/header_parser_corpus/clusterfuzz-testcase-minimized-header_parser_fuzz_test-4709439954485248.fuzz

```diff
@@ -0,0 +1,18 @@
+headers_to_add {
+  header {
+    key: "A"
+    value: "%START_TIME(%f%f%f%E92E1%f%f%E4/1f%E1%f%f%E4/1f%E46E%f%E461f%Eff%f%f%E4/0f%E1%f%f%E4/1f%E47E768E4/1f%E47E768f%\305\2231%f%f%E461f%Eff%f%E916%0f%6f%)%"
+  }
+}
+headers_to_add {
+  header {
```

Comment:
oops :P swapped which testcase to remove

Reply:
Oops, totally forgot that I had to remove the corpus entry too!